genestack_client/scripts/genestack_application_manager.py

```diff
@@ -536,6 +533,15 @@ def upload_single_file(application, file_path, version, override,
     if not no_wait:
```

Comment:
We added a message if we are waiting, but we should also add a message if we are not waiting.

If we are not waiting, we should remind users that if JAR was successfully loaded then there could be other errors and warnings that must be checked using `status` command manually by user.